[2023-12-13 19:50:37,697] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: dag_with_python_weather_v01.fill 2023-12-13T17:00:00+00:00 [queued]>
[2023-12-13 19:50:39,417] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: dag_with_python_weather_v01.fill 2023-12-13T17:00:00+00:00 [queued]>
[2023-12-13 19:50:39,827] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2023-12-13 19:50:39,832] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2023-12-13 19:50:39,861] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2023-12-13 19:50:40,447] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): fill> on 2023-12-13T17:00:00+00:00
[2023-12-13 19:50:40,877] {standard_task_runner.py:52} INFO - Started process 1081 to run task
[2023-12-13 19:50:41,011] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'dag_with_python_weather_v01', 'fill', '2023-12-13T17:00:00+00:00', '--job-id', '36', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_python_weather.py', '--cfg-path', '/tmp/tmpu887yxme', '--error-file', '/tmp/tmpcvc7ei85']
[2023-12-13 19:50:41,142] {standard_task_runner.py:77} INFO - Job 36: Subtask fill
[2023-12-13 19:50:41,901] {logging_mixin.py:104} INFO - Running <TaskInstance: dag_with_python_weather_v01.fill 2023-12-13T17:00:00+00:00 [running]> on host 5f3722f1fc3b
[2023-12-13 19:50:43,507] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=ggoni
AIRFLOW_CTX_DAG_ID=dag_with_python_weather_v01
AIRFLOW_CTX_TASK_ID=fill
AIRFLOW_CTX_EXECUTION_DATE=2023-12-13T17:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-12-13T17:00:00+00:00
[2023-12-13 19:50:47,133] {python.py:118} INFO - Done. Returned value was: None
[2023-12-13 19:50:47,230] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=dag_with_python_weather_v01, task_id=fill, execution_date=20231213T170000, start_date=20231213T195037, end_date=20231213T195047
[2023-12-13 19:50:47,515] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-12-13 19:50:47,607] {local_task_job.py:146} INFO - Task exited with return code 0
